test/controllers/hrs_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```

Comment:
Why this file added with this commit have you worked on this ?

Reply:
No,I did not work.